pom.xml

```diff
@@ -322,14 +322,6 @@
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
                                     <resource>io/usethesource/vallang/type/types.config</resource>
                                 </transformer>
-                                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
```

Comment:
Why did hmyhey get removed? Shouldn't they still be there, but in a new directory?

Reply:
I tried to remove them for the sake of minimalism. That worked. I guess the rewrites make sure the collisions don't happen anymore.